lib/slack.ex

```diff
@@ -171,6 +171,20 @@ defmodule Slack do
   end
 
   @doc """
+  Sends `text` and `attachments` to `channel` for the given `slack` connection.
+  """
+  def send_attachments(text, attachments, channel, slack) do
```

Comment:
So I think we have two options, one where we accept a single attachment and pass through to `send_attachments` or raise. I think both make sense, but a nicer API *might* be accepting non-list attachment, wrapping it in `[]`, and calling itself.

ex:

```
def send_attachments(text, attachment, channel, slack) do
  send_attachment(text, [attachment], channel, slack)
end
```